src/modeler/AutoRest.Swagger/Validation/LROStatusCodesValidation.cs

```diff
@@ -26,6 +26,11 @@ public class LROStatusCodesValidation : TypedRule<Dictionary<string, Dictionary<
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of change implementing this rule can cause.
+        /// </summary>
+        public override ValidationChangesImpact ValidationChangesImpact => ValidationChangesImpact.ServiceImpactingChanges;
```

Comment:
this is most likely SDKImpacting, I believe, since it's the way we ask folks to model the long running operation in the swagger, including the "end result" model, so that our generated code knows what to expect after polling. Thoughts?